glance/files/mitaka/glance-api.conf.Debian

```diff
@@ -799,55 +799,101 @@ stores = {{ server.storage.engine }}
 # (dict value)
 #http_proxy_information =
 
+{%- if 'swift' in storage_engines %}
```

Comment:
i named it swift_store to be in sync with the naming within glance. but if you prefer it to be swift:store, instead of swift_store, i can update it to that. Let me know.